SonarTfsAnnotate/Program.cs

```diff
@@ -73,16 +73,20 @@ static int Main(string[] args)
                         continue;
                     }
 
+                    WorkspaceInfo workspaceInfo = Workstation.Current.GetLocalWorkspaceInfo(path);
```

Comment:
Since we get the workspace info from path, unless the current worksStation cache has this mapping info, this statement will return null. So, if the first item to annotate is not in mapping, we still wont be able to get the workspace info to update the cache.